installer/vanilla.porter.yaml

```
@@ -88,6 +95,12 @@ mixins:
   - kubernetes
 
 install:
+  - exec:
+      description: "Set custom namespace for operator if present"
```

```suggestion
      description: "Set custom namespace for operator
```

The parameter `operatorNamespace` will always be populated. So if it's not customized by the user it will just have the default value. Since this step always runs and the helper script always runs kustomize, let's make it clear that this logic always runs.